cmake/CompilationConfigure.cmake

```diff
@@ -509,6 +509,10 @@ if (LINUX)
     "Whether to enable the port mapping isolator."
     FALSE)
 
+  if (ENABLE_JEMALLOC_ALLOCATOR)
+    add_definitions(-DENABLE_JEMALLOC_ALLOCATOR)
```

Comment:
As per: https://cmake.org/cmake/help/v3.8/command/add_definitions.html?highlight=i

Reply:
Thanks! Any reason you don't have it further up next to the option definition? Similar to how xfs's option and if condition are next to each other?